habitat-hitl/habitat_hitl/environment/controllers/gui_controller.py

```diff
@@ -161,18 +269,25 @@ def act(self, obs, env):
                     base_k: base_action,
                 }
             )
-        if arm_action is not None:
-            action_names.append(arm_name)
-            action_args.update(
-                {
-                    arm_k: arm_action,
-                    grip_k: grasp,
-                }
-            )
+        # if arm_action is not None:
+        #     action_names.append(arm_name)
+        #     action_args.update(
+        #         {
+        #             arm_k: arm_action,
+        #             grip_k: grasp,
+        #         }
+        #     )
         if len(action_names) == 0:
             raise ValueError("No active actions for human controller.")
 
-        return {"action": action_names, "action_args": action_args}
+        assert len(base_action) == self._num_base_vel_actions
```

Comment:
Could we rename `base_action` and type it? The current name doesn't convey that it can sometimes be a collection structure. This also makes it difficult to understand why this assert would break.

Reply:
I didn't rename or type this, but I simplified the code. It is always a numpy array. The assert would break if _num_base_vel_actions (passed in to `GuiRobotController.__init__`) doesn't match the base_velocity action space we find here in `act()`.